peer_mentoring/tests/fixtures.py

```diff
@@ -0,0 +1,99 @@
+import io
+
+import pytest
+from django.core.files.base import ContentFile
+from django.test import Client
+from factories import UserFactory
+from groups.models import Avatar, Category, Comment, Group, GroupJoinRequest, Post
+from PIL import Image
+
+# from groups.views import group_index, groups_moderated
+
+
+"""
+Fixtures
+"""
+
+
+# Fixture creates an avatar instance for group fixture
+@pytest.fixture
+def avatar(db):
+    avatar = Avatar(name="Test Avatar")
+    img = Image.new("RGB", (60, 30), color=(73, 109, 137))  # creating a dummy image
+    img_file = io.BytesIO()
+    img.save(img_file, format="JPEG")
+    img_file.name = "test_image.jpg"
+    img_file.seek(0)
+    avatar.image.save(img_file.name, ContentFile(img_file.read()))
+    avatar.save()
+    return avatar
+
+
+# Fixture creates a category instance for group
+@pytest.fixture
+def category(db):
+    return Category.objects.create(name="Test Category")
+
+
+# Creates user instance
+@pytest.fixture
+def user(db):
+    return UserFactory()
+
+
+@pytest.fixture
+def user2(db):
+    return UserFactory.create()
+
+
+# fixture handles client authentication
+@pytest.fixture
+def client_authenticated(user):
```

Comment:
pytest-django already has this fixture I believe

Reply:
I'll look back at the docs.

Reply:
Oh yeah, just the client fixture